tests/unit/dataactbroker/test_account_handler.py

```diff
@@ -99,4 +103,115 @@ def test_max_login_failure(monkeypatch):
                     "Please contact DATABroker@fiscal.treasury.gov to obtain access."
 
     # Not in cgac group
-    assert error_message == json.loads(json_response.get_data().decode("utf-8"))['message']
\ No newline at end of file
+    assert error_message == json.loads(json_response.get_data().decode("utf-8"))['message']
+
+
+def test_list_submissions_success(database, job_constants, monkeypatch):
```

Comment:
Would it make sense to split this into multiple tests? Then each test could be named to describe the situation it's testing a bit more accurately and you wouldn't need to clean up after every set of assertions.